[USING] explain bip 69 (#867)
zkSNACKs/WasabiDoc: docs/using-wasabi/BIPs.md

:::details
### BIP 69: Lexicographical Indexing of Transaction Inputs and Outputs

[BIP 69: Lexicographical Indexing of Transaction Inputs and Outputs](https://github.com/bitcoin/bips/blob/master/bip-0069.mediawiki) is a standard to sort inputs and outputs deterministically.
This was intended to improve privacy, but it only does if all the wallets use this standard.

> Because only a minority of wallets do however, this leads to an easy wallet fingerprint.

Reviewer: ```suggestion
Because only a small number of wallets do that, this leads to an easy wallet fingerprint.
```

> This was intended to improve privacy, but it only does if all the wallets use this standard.
> Because only a minority of wallets do however, this leads to an easy wallet fingerprint.

Author: ```suggestion
This was intended to improve privacy, but it only does so if all wallets use this standard.
Because only a small number of wallets do use it however, this leads to an easy wallet fingerprint.
```
